Source/arcadia/internal/compiler.clj

```diff
@@ -33,9 +33,11 @@
 (defn aot [path ns-syms]
   "aot ns-syms to the given path, with all dependencies"
   (Arcadia.Boot/SetClojureLoadPath)
+  (Arcadia.Boot/AddSourcePaths)
   (aot-namespaces path (concat ns-syms [
     'clojure.core
     'clojure.core.server
     'arcadia.internal.namespace
     'arcadia.repl]))
-  (Arcadia.Boot/SetClojureLoadPathWithDLLs))
\ No newline at end of file
+  (Arcadia.Boot/SetClojureLoadPathWithDLLs)
+  (Arcadia.Boot/AddSourcePaths))
```

Comment:
This needs to be added twice?

Reply:
Oh I get it, because of `aot-namespaces`.

Reply:
yeah a bit weird but it's just removing `/dlls` from the load path before compilation so it doesn't skip any namespaces that were compiled